wildmatch_test.go

```diff
@@ -597,6 +597,35 @@ var Cases = []*Case{
 		Subject: `foo-a.txt`,
 		Match:   false,
 	},
+	{
```

Comment:
It might be good to have a test case where `*.txt` matches a top-level path like `file.txt` with `Basename` set